### DISCUSSION TITLE
Change blockchain location.

### DISCUSSION THREAD

**Discussion by ZbyszekM** [guydavis/machinaris#979] [Q&A] [answered]
Hi there!
I'm trying to find a way to change  the default chia DB location inside Machinaris container. Is there any simple solution like it used to be in chia via config file or is it better to map a external drive in docker-compose.yml?

PS didn`t find similar topic, discord invitation does not work neither.

**Comment by guydavis** [comment, marked as answer]
Yes Chia's config yaml exists.  However, I would recommend changing the host path for the Docker volume mapped to /root/.chia inside the container instead. This let's you run Machinaris off a different drive/path for example.

**Reply by ZbyszekM** [reply]
Yes, it works!
Thank you!!
